code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/PropositionsRepositoryImpl.kt

```diff
@@ -0,0 +1,55 @@
+/*
+  Copyright 2024 Adobe. All rights reserved.
+  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License. You may obtain a copy
+  of the License at http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software distributed under
+  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+  OF ANY KIND, either express or implied. See the License for the specific language
+  governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.optimize
+
+import com.adobe.marketing.mobile.Event
+import com.adobe.marketing.mobile.MobileCore
+import com.adobe.marketing.mobile.services.Log
+
+object PropositionsRepositoryImpl : PropositionsRepository {
+    private val SELF_TAG: String = "PropositionsDisplayRepositoryImpl"
+
+    override fun trackTapInteraction(xdm: Map<String, Any>) {
```

Comment:
We decided not to support batched tap interactions right? Please remove this

Reply:
removed @spoorthipujariadobe